index.html

```diff
@@ -1264,11 +1264,42 @@
       </section>
 
       <section>
-        <h3>Storage Location of Claims</h3>
+        <h3>Storage Providers and Data Mining</h3>
 
-        <p class="issue">
-Storage location of claims may mine data or forward data.
+        <p>
+When a holder receives a claim from an issuer, it is highly likely that the
+claim will need to be stored somewhere (e.g. in a credential repository).
+Holders are warned that the information in a verifiable claim is often
```

Comment:
Generally speaking "often" can be replaced with "may" to avoid having to back up these sorts of declarations.